bigframes/ml/preprocessing.py

```diff
@@ -356,11 +368,16 @@ def _parse_from_sql(cls, sql: str) -> tuple[KBinsDiscretizer, str]:
 
         Returns:
             tuple(KBinsDiscretizer, column_label)"""
-        s = sql[sql.find("(") + 1 : sql.find(")")]
-        array_split_points = s[s.find("[") + 1 : s.find("]")]
-        col_label = s[: s.find(",")]
-        n_bins = array_split_points.count(",") + 2
-        return cls(n_bins, "uniform"), col_label
+        if sql.startswith("ML.QUANTILE_BUCKETIZE"):
+            s = sql[sql.find("(") + 1 : sql.find(")")]
+            col_label, num_bins = s.split(", ")
```

Comment:
split by ", " is kinda dangerous. What if the SQL doesn't contain a space. 

What you actually want is split(",") with .strip on each str.

Reply:
```s = sql[sql.find("(") + 1 : sql.find(")")]```
Can be shared.

Reply:
Done.